addons/mail/static/src/components/discuss_sidebar_item/discuss_sidebar_item.xml

```diff
@@ -28,6 +28,9 @@
                     <div class="o_DiscussSidebarItem_item o_DiscussSidebarItem_name" t-att-class="{ 'o-item-unread': thread.message_unread_counter > 0 }">
                         <t t-esc="thread.displayName"/>
                     </div>
+                    <t t-if="thread.mass_mailing">
+                        <i class="fa fa-envelope-o" title="Sends messages by email" role="img" aria-label="Send by mail"/>
```

Comment:
no aria-label when title says the same thing
Also that was the title we had before? Feels strange

Reply:
Yes, I purely copy/pasted it from the old template

Reply:
Ok then, let's keep the text like that for now